apps/website/get-started/developing/ui.md

```diff
@@ -0,0 +1,21 @@
+---
+title: UI (@clr/ui)
+toc: true
+---
+
+If you want to use the HTML/CSS implementations, you need to get the latest code from NPM and then add the assets to your project.
```

Comment:
I don't want to promote this at all, its not the future or a good recommendation anymore with Core being available. Let's remove this.

Reply:
FYI @yuxin-ux: I'm going to be removing the HTML/CSS content.